rebasehelper/utils.py

```diff
@@ -323,7 +322,26 @@ def close(self):
         session = requests.Session()
         session.mount('ftp://', FTPAdapter())
 
-        r = session.get(url, stream=True)
+        try:
+            return session.get(url, **kwargs)
+        except requests.exceptions.ConnectionError as e:
+            logger.debug(e)
```

Comment:
You should probably prepend the error message with some context, at least the exception name. Also, you are passing exception instance to a method which expects string.